server/middlewares/validateUserId.js

```diff
@@ -0,0 +1,13 @@
+const validateUserId = (req, res, next) => {
```

Comment:
What will this middleware do?

Reply:
Validates that the id passed as a params is the same as that of the loggedIn  user. Does this check before the request hits the controller.